### PR TITLE
Clarify how to set up a private broker with TLS

### PR DESCRIPTION
**Pull request by kw217**
Shuffle paragraphs around so the instructions are in the right place, and be more explicit about exactly what to set where.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/mqtt.markdown [hunk 1]
```diff
@@ -50,28 +50,33 @@ The most private option is running your own MQTT broker.
...
-### Connect to a public broker
+### Use a public broker
 
 The Mosquitto project runs a [public broker](https://test.mosquitto.org). This is the easiest to set up, but there is no privacy as all messages are public. Use this only for testing purposes and not for real tracking of your devices or controlling your home. To use the public mosquitto broker, configure the MQTT integration to connect to broker `test.mosquitto.org` on port 1883 or 8883.
 
-<div class='note'>
+## Broker configuration
 
-If you experience an error message like `Failed to connect due to exception: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed`, then add `certificate: auto` to your broker configuration and restart Home Assistant.
+MQTT broker settings are configured when the MQTT integration is first setup, and can be changed later if needed.
```
Comment (frenck) [inline source/_integrations/mqtt.markdown]:
```suggestion
MQTT broker settings are configured when the MQTT integration is first set up and can be changed later if needed.
```

File: source/_integrations/mqtt.markdown [hunk 2]
```diff
@@ -50,28 +50,33 @@ The most private option is running your own MQTT broker.
...
 
-</div>
+Add the MQTT integration, then provide your broker's hostname (or IP address) and port, and (if required) the username and password that Home Assistant should use. To change the settings later, click on "Configure" in the integration page in the UI, then "Re-configure MQTT".
```
Comment (frenck) [inline source/_integrations/mqtt.markdown]:
```suggestion
Add the MQTT integration, then provide your broker's hostname (or IP address) and port and (if required) the username and password that Home Assistant should use. To change the settings later, click on "Configure" on the integration page in the UI, then "Re-configure MQTT".
```

**Review by frenck — APPROVED** [review]
Thanks, @kw217 👍

File: source/_integrations/mqtt.markdown [hunk 3]
```diff
@@ -50,28 +50,33 @@ The most private option is running your own MQTT broker.
...
-</div>
+Add the MQTT integration, then provide your broker's hostname (or IP address) and port and (if required) the username and password that Home Assistant should use. To change the settings later, click on "Configure" on the integration page in the UI, then "Re-configure MQTT".
 
-## Broker configuration
+<div class='note'>
 
-MQTT broker settings are configured when the MQTT integration is first setup, and can be changed if needed. To change the settings, click on "Configure" in the integration page in the UI, then "Re-configure MQTT".
+If you experience an error message like `Failed to connect due to exception: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed`, then add
+```yaml
+mqtt:
```
Comment (jbouwh) [inline source/_integrations/mqtt.markdown:74]: Setting this using YAML is deprecated, the UI setting should be used instead.

Comment (jbouwh) [reply source/_integrations/mqtt.markdown:74]: See https://github.com/home-assistant/home-assistant.io/pull/25538 for a correction on this.

**Pull request merged**